internal/vercheck/vercheck.go

```diff
@@ -136,6 +181,23 @@ func isNewLauncherAvailable() bool {
 	return true
 }
 
+func isNewBinaryAvailable() bool {
+
+	availableVersion, err := getAvailableVersion()
+	if err != nil {
+		debug.Log("ERROR: failed to get available version for CLI: %v\n", err)
+		return false
+	}
+
+	// If currentVersion is invalid, semver.Compare will return 0 and we'll assume
+	// that a new binary is not available.
+	if semver.Compare(currentBinaryVersion, availableVersion) >= 0 {
```

Comment:
We might want to wrap semver.Compare in a function that optionally adds `v` to all versions if they are missing. The versions we save in the files, and what we use for DEVBOX_USE_VERSION are all without v.

Reply:
did it in #965